nevergrad/functions/images/core.py

```diff
@@ -246,19 +248,22 @@ def make_folder_functions(
         assert model in {"resnet50", "test"}
         tags = {"folder": "#FAKE#" if folder is None else Path(folder).name, "model": model}
         classifier: tp.Any = Resnet50() if model == "resnet50" else TestClassifier()
-        imsize = 224
-        transform = tr.Compose([tr.Resize(imsize), tr.CenterCrop(imsize), tr.ToTensor()])
+        classifier.eval()
+
+        transform = tr.Compose([tr.Resize(256), tr.CenterCrop(224), tr.ToTensor()])
         if folder is None:
             x = torch.zeros(1, 3, 224, 224)
             _, pred = torch.max(classifier(x), axis=1)
             data_loader: tp.Iterable[tp.Tuple[tp.Any, tp.Any]] = [(x, pred)]
         elif Path(folder).is_dir():
-            ifolder = torchvision.datasets.ImageFolder(folder, transform)
-            data_loader = torch.utils.DataLoader(
+            print(f"Folder  = {folder}")
```

Comment:
is this needed? using a logger would be better but if that helps for now lets keep in